deep-learning/_posts/2017-08-24-sgans.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: review
+title: "Stacked Generative Adversarial Networks"
+tags: deep-learning GAN CNN
+author: "Philippe Poulin"
+cite:
+    authors: "X. Huang, Y. Li, O. Poursaeed, J. Hopcroft, S. Belongie"
+    title:   "Stacked Generative Adversarial Networks"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1612.04357.pdf"
+---
+   
+## Summary
+
+Conditional GANs are trained to hierarchically generate an encoder's internal representations by first conditioning on the class label (or encoding), down to generating the input image.
+
+![](/deep-learning/images/sgan/figure1.png)
+
+There are 3 combined types of losses:
+
+- Adversarial loss: Each GAN-generated representation should be able to fool a discriminator when compared to the encoder-generated representations (purple arrows)
+- Conditional loss: An encoder should produce the same higher representation whether its input was generated by a GAN or an encoder (blue arrows)
+- Entropy loss: A new deep model `Q_i` (which shares parameters with `D_i`) tries to predict the GAN input noise given the GAN's output `Q_i( z_i | \hat{h}_i )` (green arrows)
```

Comment:
wouldn't it be better to use the mathematical formulation $$Q_i$$  ?

Reply:
Thanks, I thought latex math equations weren't supported in Github Markdown